acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java

```diff
@@ -66,6 +66,8 @@ public class Signer extends FilecoinJsonRpcEndpoint {
   public static final String ETH2_PUBLIC_KEYS = "/api/v1/eth2/publicKeys"; // bls keys
   public static final String RELOAD_ENDPOINT = "/reload";
 
+  public static final String GENERIC_SIGN_EXT_ENDPOINT = "/eth/v1/ext/sign/{identifier}";
```

Comment:
Does this need its own endpoint? Thought it would be under the ETH2_SIGN_ENDPOINT but using a different domain. Or prefixed with `/api/v1/eth2/` path so it is consistent with other endpoints.